frontend/src/app/api/SearchOpportunityAPI.ts

```diff
@@ -79,16 +80,26 @@ export default class SearchOpportunityAPI extends BaseApi {
   private buildPagination(
     searchInputs: SearchFetcherProps,
   ): PaginationRequestBody {
-    const { sortby, page } = searchInputs;
+    const { sortby, page, fieldChanged } = searchInputs;
+
+    // When performing an update (query, filter, sortby change) - we want to
+    // start back at the 1st page (we never want to retain the current page).
+    // In addition to this statement - on the client (handleSubmit in useSearchFormState), we
+    // clear the page query param and set the page back to 1.
+    // On initial load (SearchFetcherActionType.InitialLoad) we honor the page the user sent. There is validation guards
+    // in convertSearchParamstoProperTypes keep 1<= page <= max_possible_page
+    const page_offset =
+      searchInputs.actionType === SearchFetcherActionType.Update &&
+      fieldChanged !== "pagination"
+        ? 1
+        : page;
```

Comment:
when making the actual API call , and we're in the scenario where we've changed something that's not pagination - call the API on the 1st page of results